src/utils/startup.js

```diff
@@ -8,7 +8,10 @@ import orgs from '@/api/orgs'
 import { getPropView, isViewHasOrgs } from '@/utils/jms'
 
 const whiteList = ['/login', process.env.VUE_APP_LOGIN_PATH] // no redirect whitelist
-const autoEnterOrgs = ['00000000-0000-0000-0000-000000000001', '00000000-0000-0000-0000-000000000000']
+const autoEnterOrgs = [
+  '00000000-0000-0000-0000-000000000001',
+  '00000000-0000-0000-0000-000000000000'
+]
 
 function reject(msg) {
   return new Promise((resolve, reject) => reject(msg))
```

Comment:
There does not seem to be any code differences between this snippet and the original snippet. The functions reject(), getOrgInfoByGuid() and addPermissionToRole are present but their implementation differs slightly from what you provided earlier in the code. Additionally, there's only one line of text left over on the file that I cannot identify its purpose.

It also appears like an extra blank lines have been added at the end which do not impact functionality nor the overall structure. 

Overall, if it were me making these changes, I might go further by using proper documentation/ commenting better instead.

Code diff suggestion: Review comments where necessary, document methods thoroughly.

Also consider removing any unused imports / variables.